### PR TITLE
update pypi to elasticdl-client in README

### PR DESCRIPTION
**Pull request by skydoorkai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,11 @@
 [![Travis-CI Build Status](https://travis-ci.com/sql-machine-learning/elasticdl.svg?branch=develop)](https://travis-ci.com/sql-machine-learning/elasticdl)
 [![Code Coverage](https://codecov.io/gh/sql-machine-learning/elasticdl/branch/develop/graph/badge.svg)](https://codecov.io/gh/sql-machine-learning/elasticdl)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
-[![PyPI Status Badge](https://badge.fury.io/py/elasticdl.svg)](https://pypi.org/project/elasticdl/)
+[![PyPaI Status Badge](https://badge.fury.io/py/elasticdl-client.svg)](https://pypi.org/project/elasticdl-client/)
 
 ElasticDL is a Kubernetes-native deep learning framework built on top of
 TensorFlow 2.0 that supports fault-tolerance and elastic scheduling.
 
-|                          | TensorFlow 1.x graph mode | TensorFlow 2.x eager execution |
```
Comment (brightcoder01) [inline README.md:11]: Why delete this section?

File: README.md [hunk 2]
```diff
@@ -34,7 +25,7 @@ graph execution process.
 
 ### Minimalism Interface
 
-Given a model defined with Keras API, train the model with a command line.
+Given a [model](model_zoo/mnist_functional_api/mnist_functional_api.py) defined with Keras API, train the model distributedly with a command line.
```
Comment (brightcoder01) [inline README.md]: distributedly -> in a distributed way

File: README.md (hunk 1 above)
Comment (skydoorkai) [reply README.md:11]: Uber Horovod  表述不太对， 也是支持 2.x. 感觉没有必要放。

还有我们现在不算是 early-stage了。

File: README.md (hunk 2 above)
Comment (skydoorkai) [reply README.md]: `distributedly` is more concise.

**Review by brightcoder01 — APPROVED** [review]

**Pull request merged**